### PR TITLE
master LRQA-76554 | Update tabbing and spacing in headless poshi tests

### PR DESCRIPTION
**Pull request by yanan**
Background:
Update tabbing and spacing in request body for headless tests

Test Plan
Updating tabbing and spacing format for '''{}''' body part in headless poshi tests:
1. HeadlessDiscovery.testcase
2. BlogPostingAPI.macro
3. ObjectDefinitionAPI.macro

JIRA Link:
https://issues.liferay.com/browse/LRQA-76554

### CODE REVIEW

**Comment by yanan** [comment]
ci:test:headless-frontend-functional:nocompile

**Comment by yanan** [comment]
Hello @ling-alan-huang, I've updated the body part in this pr, please let me know if anything needs to be updated, thanks :D

File: portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectDefinitionAPI.macro [hunk 1]
```diff
@@ -374,10 +369,9 @@ definition {
 			-u test@liferay.com:test \
 			-H Content-Type: application/json \
 			-H accept: application/json
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectDefinitionAPI.macro:371]: maybe we can add `\` in the end of this line, just to be consistent

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDiscovery/HeadlessDiscovery.testcase [hunk 2]
```diff
@@ -369,10 +367,7 @@ definition {
 			var siteId = JSONGroupAPI._getGroupIdByNameNoSelenium(
 				groupName = "Guest",
 				site = "true");
-			var requestBody = '''{
-                "contentStructureId": ${ddmStructureId},
-                "title": "title of structured content"
-            }''';
+			var requestBody = '''{"contentStructureId": ${ddmStructureId},"title": "title of structured content"}''';
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDiscovery/HeadlessDiscovery.testcase:370]: aligning it is a nice idea, but I think we lose readability this way, is it possible to make it one `key:value` per line?

**Pull request closed without merging**